.github/workflows/codeql-analysis.yml

```diff
@@ -80,7 +80,7 @@ jobs:
           mvn -e -B -V org.apache.maven.plugins:maven-wrapper-plugin:3.1.0:wrapper "-Dmaven=3.8.6"
 
       - name: Clean install dependencies and build
-        run: mvn clean install -DskipTests -B -V -Pfast -Dmaven.javadoc.skip=true -T4
+        run: mvn clean install -DskipTests -B -V -Pfast -Dmaven.javadoc.skip=true -Dorg.slf4j.simpleLogger.dateTimeFormat=HH:mm:ss,SSS -Dorg.slf4j.simpleLogger.showDateTime=true
```

Comment:
We should add javadoc.skip to the `fast` profile.

Reply:
Good idea. However, I'm not seeing any improvements on Github's end (still over 17 minutes for that command). Currently debugging to see what's going on